docker-compose.yml

```
@@ -1,12 +1,31 @@
 version: '3.1'
 
 services:
+
   db:
-    container_name: postgresovisco
-    image: postgres:15.1
+    container_name: postgres
```

niech zostanie postgresovisco
```suggestion
    container_name: postgresovisco
```